### PR TITLE
Feature/program landing page non authenticated

### PR DESCRIPTION
**Pull request by mark-escosura**
## Description

Created and Styled Landing Programs Page (WIP) 

`Co-Authored`: @MaryAngelique  @Raj-04 

## Loom Video

[Loom Video](https://www.loom.com/share/6833f0d3850b4ca8a9fadc2c980c9c21)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by mark-escosura** [comment]
[As a non-authenticated user, I can view the non-authenticated Program Landing Page](https://trello.com/c/ZnpS8V2C/213-as-a-non-authenticated-user-i-can-view-the-non-authenticated-program-landing-page)

**Review by Adamh1990 — APPROVED** [review]
Other than a few little things that look extra or could maybe use some explaining, the final product looks fantastic, well done!

File: src/components/pages/LandingPrograms/CoderCamp.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+              </div>
+            </div>
+          </div>
+        </div>
+
+        {/* Testimonials */}
+        {/* Testimonials */}
+        <div className="pl__na__testimonials__container">
+          <div>
+            <LeftOutlined style={{ color: '#fefefe', fontSize: '62px' }} />
```
Comment (Raj-04) [inline src/components/pages/LandingPrograms/CoderCamp.js:83]: We could have ignored the inline styling over here, and done it with the other files for styling. To stay persistent with the format

File: src/components/pages/LandingPrograms/CoderCamp.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            <div className="pl__testimony__section__two">
+              <Text className="pl__na__testimony__name__container">
+                David
+                <br />
+                CoderHeroes Parent
+              </Text>
+            </div>
+          </div>
+          <div>
+            <RightOutlined style={{ color: '#fefefe', fontSize: '62px' }} />
```
Comment (Raj-04) [inline src/components/pages/LandingPrograms/CoderCamp.js:115]: The same thing over here could be added to the stylings page.

**Review by Raj-04 — APPROVED** [review]
Overall, great work, really good naming conventions for the classname, really import for styling. Done a great job for the landing page.

**Review by MaryAngelique — APPROVED** [review]

File: src/styles/ProgramsLandingStyles/yoga.less [hunk 3]
```diff
@@ -0,0 +1,109 @@
+// <----- CoderYoga ----->
+
```
Comment (moua0061) [inline src/styles/ProgramsLandingStyles/yoga.less:2]: I like how you styled this using divs to separate all your containers and wrapping them correctly to display the awesome design.

**Review by moua0061 — APPROVED** [review]

File: src/components/pages/LandingPrograms/CoderCamp.js (hunk 2 above)
Comment (sagun1151) [reply src/components/pages/LandingPrograms/CoderCamp.js:115]: Agree with Raj keep everything in the same format in terms of styling.

File: src/components/pages/LandingPrograms/CoderSitters.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+              Our lessons include playing age-appropriate
+              <br />
+              coding games online, in tandem with a wide
+              <br />
+              range of interactive offline activities. These
+              <br />
+              might include: drawing, prototyping, crafting
+              <br />
+              or imaginative brainstorming
+            </Text>
```
Comment (sagun1151) [inline src/components/pages/LandingPrograms/CoderSitters.js:39]: Using <br> for line breaks is great

**Review by sagun1151 — APPROVED** [review]

File: src/styles/ProgramsLandingStyles/yoga.less [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+      width: 15rem;
+      height: 3.5rem;
+      border: none;
+      border-radius: @btn__border-radius;
+      box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
+      transition: 0.15s ease-in;
+      cursor: pointer;
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (mark-escosura) [reply src/styles/ProgramsLandingStyles/yoga.less:109]: Sorry about that. 
pl stands for: program landing
na stands for: non-authorized
but I will leave comments to what those stands for in the future! Thank you!

File: src/components/pages/LandingPrograms/CoderCamp.js (hunk 1 above)
Comment (mark-escosura) [reply src/components/pages/LandingPrograms/CoderCamp.js:83]: I will make these changes! You're not wrong! Thank you Raj!

File: src/components/pages/LandingPrograms/CoderSitters.js (hunk 4 above)
Comment (mark-escosura) [reply src/components/pages/LandingPrograms/CoderSitters.js:39]: line breaks are awesome. It is useful very useful on areas, where the division of lines is significant!

**Pull request merged**